cmd/gopcomm/community_yap.gox

```diff
@@ -1,10 +1,27 @@
+import "github.com/goplus/community/cmd/gopcomm/internal/config"
```

Comment:
internal pkg can be here: "github.com/goplus/community/internal/config"

Reply:
Thanks, I will move these packages.